assets/scss/core/_reset.scss

```diff
@@ -25,6 +25,22 @@ RESET DOPPLER STYLE GUIDES
   box-sizing: border-box;
 }
 
+::-webkit-scrollbar-thumb {
+  background: rgba(0, 0, 0, 0.3);
```

Comment:
This color maybe should be set as a variable right?

Reply:
Esto hasta quizás se modifique y no lo vamos a usar en ningún otro lado. Por el momento lo dejaría asi.